x-pack/plugins/observability_solution/ux/public/components/app/rum_dashboard/visitor_breakdown_map/embedded_map.test.tsx

```diff
@@ -26,19 +25,13 @@ describe('Embedded Map', () => {
   });
 });
 
-const mockEmbeddable = embeddablePluginMock.createStartContract();
-
-mockEmbeddable.getEmbeddableFactory = jest.fn().mockImplementation(() => ({
-  create: () => ({
-    reload: jest.fn(),
-    setRenderTooltipContent: jest.fn(),
-    setLayerList: jest.fn(),
-  }),
-}));
+const mockMapsApi = {
```

Comment:
Same comment as in your other PR, lets rename as `mockMapsStartApi` or `mockMapsStartService` to avoid naming confusion with `MapApi` type